src/releng_docs/services_shipit.rst

```diff
@@ -116,10 +116,16 @@ Initially we will integrate the legacy ship-it/release-runner infrastructure wit
 7. The pipeline service sees that S is finished, and creates step P via the Taskcluster service.
 
 
-Steps
------
+Step Services
+-------------
 
-Step Services API
+.. toctree::
+    :maxdepth 1
+
+    shipit_signoffs/shipit-signoffs
+
+
```

Comment:
I think TOC looks a bit weird for the shipit family section. We should improve this (no idea how, yet).